src/core/examples.js

```diff
@@ -6,69 +6,71 @@
 // be used by a containing shell to extract all examples.
 
 import { pub } from "core/pubsubhub";
+import "deps/hyperhtml";
 import css from "deps/text!core/css/examples.css";
 
 export const name = "core/examples";
+const isEmpty = str => /^\s*$/.test(str);
 
-function makeTitle(conf, $el, num, report) {
-  const txt = num > 0 ? " " + num : "";
-  const $tit = $(
-    `<div class='example-title'><span>${conf.l10n.example}${txt}</span></div>`
-  );
-  report.title = $el.attr("title");
-  if (report.title) {
-    $tit.append(
-      $("<span style='text-transform: none'>: " + report.title + "</span>")
-    );
-    $el.removeAttr("title");
-  }
-  $tit.addClass("marker");
-  return $tit;
+function makeTitle(conf, elem, num, report) {
+  report.title = elem.title;
+  if (report.title) elem.removeAttribute("title");
+  const number = num > 0 ? " " + num : "";
+
+  return hyperHTML`
+  <div class='marker example-title'><span>${conf.l10n.example}${number}</span>${
```

Comment:
So `example-title` certainly looks redundant. How about slightly changing its purpose:

```css
.example-title {
  text-transform: none;
}
```

```html
<div class='marker'><span>${conf.l10n.example}${number}</span>${
  report.title ? hyperHTML`<span class='example-title'>: ${report.title}</span>` : ""
}</div>
```

Reply:
We can inline the `text-transform: none;` though.